src/CellularParser.test.js

```diff
@@ -22,36 +20,70 @@ shouldParseAs.only = (input, expected) => {
   shouldParseAs(input, expected, true);
 };
 
-shouldParseAs('1 + 2 * 3',
+shouldParseAs(
+  "1 + 2 * 3",
   new OperatorExpression(
-    new NumberExpression('1'),
+    new NumberExpression("1"),
     TokenTypes.PLUS,
     new OperatorExpression(
-      new NumberExpression('2'),
+      new NumberExpression("2"),
       TokenTypes.TIMES,
-      new NumberExpression('3'),
-    ),
-  ),
+      new NumberExpression("3")
+    )
+  )
 );
 
-shouldParseAs('1 * 2 + 3',
+shouldParseAs(
+  "1 * 2 + 3",
   new OperatorExpression(
     new OperatorExpression(
-      new NumberExpression('1'),
+      new NumberExpression("1"),
       TokenTypes.TIMES,
-      new NumberExpression('2'),
+      new NumberExpression("2")
     ),
     TokenTypes.PLUS,
-    new NumberExpression('3'),
-  ),
+    new NumberExpression("3")
+  )
 );
 
-shouldParseAs('A1 + 2',
+shouldParseAs(
+  "A1 + 2",
   new OperatorExpression(
-    new NameExpression('A1'),
+    new NameExpression("A1"),
     TokenTypes.PLUS,
-    new NumberExpression('2'),
-  ),
+    new NumberExpression("2")
+  )
+);
+
+shouldParseAs(
+  "2 - 1",
+  new OperatorExpression(
+    new NumberExpression("2"),
+    TokenTypes.MINUS,
+    new NumberExpression("1")
+  )
+);
+
+shouldParseAs(
+  "1 * 2 - 3",
+  new OperatorExpression(
+    new OperatorExpression(
+      new NumberExpression("1"),
+      TokenTypes.TIMES,
+      new NumberExpression("2")
+    ),
+    TokenTypes.MINUS,
+    new NumberExpression("3")
+  )
+);
+
+shouldParseAs(
+  "2 / 1",
```

Comment:
This is good. I'd like to add at least one test that verifies precedence with `/`.